src/lib/coerce.js

```diff
@@ -86,10 +86,19 @@ exports.valObjectMeta = {
     boolean: {
         description: 'A boolean (true/false) value.',
         requiredOpts: [],
-        otherOpts: ['dflt'],
-        coerceFunction: function(v, propOut, dflt) {
-            if(v === true || v === false) propOut.set(v);
-            else propOut.set(dflt);
+        otherOpts: ['dflt', 'arrayOk'],
+        coerceFunction: function(v, propOut, dflt, opts) {
+            function isBoolean(value) {
+                return value === true || value === false;
+            }
```

Comment:
Let's make this a one-liner:
```suggestion
            const isBoolean = value => value === true || value === false;
```
Or
```suggestion
            const isBoolean = value => typeof value === "boolean";
```